index.html

```diff
@@ -59,7 +59,57 @@
             <div class="tweet-button">Button</div>
           </div>
```

Comment:
<a href="#" class="section-one-nav__tweet-button">Tweet</a>

Can be styled for a button

Reply:
I think this a link on twiiter, so it's not a button (which has an action like submitting a form). You can style an a-tag to look like a button